src/components/ListItemDetails.jsx

```diff
@@ -0,0 +1,19 @@
+import './ListItem.css';
+
+export function ListItemDetails({
+	totalPurchases,
+	lastPurchasedDate,
+	nextPurchasedDate,
+}) {
+	return (
+		<div>
+			{lastPurchasedDate ? (
+				<p>You have purchased this item {totalPurchases} times</p>
+			) : (
+				<p>You have not yet purchased this item.</p>
+			)}
+			{lastPurchasedDate && <p>Last purchased: {lastPurchasedDate}</p>}
```

Comment:
Since you already check for `lastePurchasedDate` in the ternary statement above, could this line also go there?